### ISSUE TITLE
Trade request error

### ISSUE THREAD

**Issue by EscApp2** [AuroraDAO/idex-api-docs#35]
Send "trade" request
Get error:
 {
    "error": "ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') FOR UPDATE' at line 1"
}
http://picplus.ru/img/1807/21/db76d06f.png
or sometimes
{
    "error": "Due to maintenance, this action has been temporarily disabled, try again soon"
}
http://picplus.ru/img/1807/21/97834087.png

Even if  I send all post data. I get same error 
http://picplus.ru/img/1807/21/15ef20d4.png

**Comment by fubar** [comment]
Hi @EscApp2, are you still getting the error? I believe we had a brief interruption overnight.

**Comment by EscApp2** [comment]
Yes still getting error. http://picplus.ru/img/1807/21/38a99a94.png
